release/llm_tests/serve/run_llm_serve_release_tests.py

```diff
@@ -94,6 +119,85 @@ def main(
 
             logger.info(f"Performance test results: {results}")
 
+            accelerator = llm_config.get("accelerator_type", "NoGpu")
+            # "A10" without "G" to match existing dashboard tags
+            if accelerator == "A10G":
+                accelerator = "A10"
```

Comment:
should we change our tags on db since it's new anyways? This does look hacky. Also we can fix the query.